api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxCurrDataset.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+
+package org.openmrs.module.eptsreports.reporting.library.datasets;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.openmrs.module.eptsreports.metadata.HivMetadata;
+import org.openmrs.module.eptsreports.reporting.library.cohorts.AgeCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.cohorts.CompositionCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.cohorts.EncounterCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.cohorts.GenderCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.cohorts.SqlCohortQueries;
+import org.openmrs.module.eptsreports.reporting.library.indicators.HivIndicators;
+import org.openmrs.module.reporting.ReportingConstants;
+import org.openmrs.module.reporting.cohort.definition.AgeCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.CohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.CompositionCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.EncounterCohortDefinition;
+import org.openmrs.module.reporting.cohort.definition.SqlCohortDefinition;
+import org.openmrs.module.reporting.dataset.definition.CohortIndicatorDataSetDefinition;
+import org.openmrs.module.reporting.evaluation.parameter.Mapped;
+import org.openmrs.module.reporting.evaluation.parameter.Parameter;
+import org.openmrs.module.reporting.evaluation.parameter.ParameterizableUtil;
+import org.openmrs.module.reporting.indicator.CohortIndicator;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TxCurrDataset {
+	
+	@Autowired
+	private AgeCohortQueries ageCohortQueries;
+	
+	@Autowired
+	private GenderCohortQueries genderCohortQueries;
+	
+	@Autowired
+	private SqlCohortQueries sqlCohortQueries;
+	
+	@Autowired
+	private EncounterCohortQueries encountertQueries;
+	
+	@Autowired
+	private HivMetadata hivMetadata;
+	
+	@Autowired
+	private CompositionCohortQueries compositionCohortQueries;
+	
+	@Autowired
+	private HivIndicators hivIndicators;
+	
+	public List<Parameter> getParameters() {
+		List<Parameter> parameters = new ArrayList<Parameter>();
+		parameters.add(ReportingConstants.START_DATE_PARAMETER);
+		parameters.add(ReportingConstants.END_DATE_PARAMETER);
```

Comment:
You need to include the location? they should find indicator per the laoction

Reply:
Yes. We started from a database that was on the QA server which didn't have locations configured, but I'll add the location parameter in my next few commits since we have the FGH database running on the server now.

Reply:
I've added the location param with [this commit](https://github.com/esaude/openmrs-module-eptsreports/commit/c55947163d4fb13cff1720ebc902c6bf7100c13d)